lua/plugins.lua

```diff
@@ -156,15 +156,12 @@ return {
 
   -- Status Line and Bufferline
   {
-    "glepnir/galaxyline.nvim",
+    -- "hoob3rt/lualine.nvim",
+    "Lunarvim/lualine.nvim",
     config = function()
-      require "core.galaxyline"
-      if lvim.builtin.galaxyline.on_config_done then
-        lvim.builtin.galaxyline.on_config_done(require "galaxyline")
-      end
+      require("core.lualine").setup()
     end,
-    event = "BufWinEnter",
-    disable = not lvim.builtin.galaxyline.active,
+    disable = not lvim.builtin.lualine.active,
```

Comment:
we can make it load on `BufWinEnter` so people can show off their faster `--staruptime` 🤣

Reply:
I want it as well.
However hot reload won't work this way atm.

Should we give up hot reload for lazyload?

Reply:
not at all, we will deal with the lazyload later, hot reload is more important rn